toc/rfc/accepted/rfc-draft-github-members.md

```diff
@@ -0,0 +1,34 @@
+# Meta
+[meta]: #meta
+- Name: Who is able to add and remove members of the Cloudfoundry GitHub org
+- Start Date: 2022-01-26
+- Author(s): @LeePorte
+- Status: Draft
+- RFC Pull Request: (fill in with PR link after you submit it)
+
+
+## Summary
+
+As the Cloudfoundry moves from the dojo model to an open committer model the existing central team in VMWare are no 
+longer appropriately positioned to grant and revoke access to users of the Cloudfoundry GitHub org
+
+## Problem
+
+Previously re was a central team in VMWare that undertook adding of individuals on successful completion of the dojo, 
+as this was the process for gaining committer rights on the repositories. However, as this model is no longer in use it
+presents a problem in terms of who should fulfil this role.
+
+## Proposal
+
+As defined in the [roles](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md), contributor and above roles
+should be members of the Cloudfoundry org and possibly other orgs if appropriate. As the responsibility to become a
+contributor is to make multiple contributions and be an active member of the community, it should for the Approvers and 
+Working Group Leads to make the judgement call on if an individual has met the criteria to become a contributor.
+
+The addition of raising and merging PRs to add members to the cloudfoundry and other foundation org(s) is the 
+responsibility or the working group approvers and leads.
+
+Removing members of the cloudfoundry and other foundation org(s) may have unintended consequences. Raising PRs to remove
+members of the cloudfoundry and other foundation org(s) is the responsibility or the working group approvers and leads.
+These PRs should be tagged `toc` and mention `@cloudfoundry/toc`. The TOC will then take responsibility for the merging
+or closing of the raised PRs.
```

Comment:
So removing members is the responsibility of the working group approvers and leads, if that is the case, why would the `@cloudfoundry/toc` need to be tagged?

Reply:
I was thinking Working group to raise the PR, but TOC to merge or close the PR.

Reply:
I've made some wording tweaks

Reply:
read much better, however, there is still an `or` which should be an `of` (responsibility of the...)

Reply:
actually that one is fixed its `consequences or removing members`

Reply:
Got that one and added a missing word